index.html

```diff
@@ -1,67 +1,345 @@
+                      <span>javaScript</span>
+                    </div>
+                  </li>
+                </ul>
+              </div>
+              <div class="action">
+                <div class="see-project">
+                  <button>See Project</button>
+              </div>
+            </div>
+          </div>
+          </div>
+          <div class="card-works-4">
+            <div class="snapshot-portfolio">
+              <img class="snp-p-img" src="assets/snapshot-card-4.svg" width="295" height="220" alt="snapshot-card">
+            </div>
+            <div class="left-block">
+              <div class="primary-text-3">
+                  <h4 class="tonic">Multi-post stories</h4>
+                  <div class="frame">
+                  <div class="client">
+                    <h6>CANOPY</h6>
+                  </div>
+                  <div>
+                    <img class="counter" src="assets/counter.svg" alt="counter">
+                  </div>
+                  <div class="role">
+                    <span>Back End Dev</span>
+                  </div>
+                  <div>
+                    <img class="counter" src="assets/counter.svg" alt="counter">
+                  </div>
+                  <div class="year-label">
+                    <span>2015</span>
+                  </div>
+                </div>
+              </div>
+              <div class="primary-text-4">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+              </div>
+                <div class="tags">
+                  <ul>
+                    <li>
+                      <div class="tag-html">
+                        <span>html</span>
+                      </div>
+                    </li>
+                    <li>  
+                      <div class="tag-css">
+                        <span>css</span>
+                      </div>
+                    </li>
+                    <li>
+                      <div class="tag-js">
+                        <span>javaScript</span>
+                      </div>
+                    </li>
+                  </ul>
+                </div>
+              <div class="action">
+                <div class="see-project">
+                  <button>See Project</button>
+                </div>
+              </div>
+            </div>
+          </div>
         </div>
-      </div>
-      <div class="primary-text">
-        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-      </div>
-
-    </section>
-  </main>
-
-</body>
-</html>
\ No newline at end of file
+        <section class="about-myself">
+          <div class="top">
+            <div class="primary-text-5">
+              <h1>About <br>
+              Myself</h1>
+            </div>
```

Comment:
- [ ] Please use a `<hx>` tag here. You should only have one `<h1>` tag in your documents to maintain a good structure and document flow. It is also good for SEO 👍